app/views/home/_about.html.erb

```diff
@@ -0,0 +1,51 @@
+<section class="container">
+  <h1 class="text-center banner-top">BrightCoders</h1>
+  <div class="text-center"><span class="banner-bottom">for a brighter future!</span></div>
+  <div class="row flex-sm-row mt-4 mx-3">
+    <div class="col-lg-4 col-md-6 p-2">
+      <div class="card">
+        <div class="p-3 text-center"><%= image_tag('icon-cap', class: 'fit-image-first card-img-top', alt:"icon cap") %></div>
+        <div class="card-body">
+          <h5 class="title">¿Qué somos?</h5>
+          <p class="text">
+            Somos una aceleradora de talento que brinda a jóvenes con iniciativa y ganas de aprender programas de
+            certificación diseñados para potenciar sus habilidades, mejorar sus posibilidades de acceso a un empleo de
+            calidad y condiciones de vida.
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <div class="col-lg-4 col-md-6 p-2">
+      <div class="card">
+        <div class="p-3 text-center">
+          <%= image_tag('icon-computer', class: 'fit-image-first card-img-top', alt:"icon cap") %>
+        </div>
+        <div class="card-body">
+          <h5 class="title">¿Qué ofrecemos?</h5>
+          <p class="text">
+            Una experiencia de aprendizaje única que te permitirá iniciar una carrera en tecnología gracias a la
+            capacitación y experiencia que lograrás colaborando en proyectos de desarrollo software reales y el
+            acompañamiento de una comunidad de mentores experimentados.
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <div class="col-lg-4 col-md-6 p-2 mx-auto">
+      <div class="card">
+        <div class="p-3 text-center">
+          <%= image_tag('icon-lamp', class: 'fit-image-third card-img-top', alt:"icon cap") %>
+        </div>
+        <div class="card-body">
+          <h5 class="title">BrightCoders</h5>
+          <p class="card-text">
+            BrightCoders Academy capacita a estudiantes, egresados y jóvenes con ganas de aprender y con conocimientos
+            intermedios de programación orientada a objetos en HTML, CCS, y JS para iniciar una carrera en tecnología.
+            También es una opción para acreditar las prácticas o estancias profesionales.
+          </p>
+        </div>
+      </div>
+    </div>
```

Comment:
I believe that you can use the Rails capacity, to reduce the use of code, but provide the same structure, using i18n Internationalization.

```suggestion
  <% t('brightcoders.description_section.items').each do |item| %>
    <div class="col-lg-4 col-md-6 p-2 mx-auto">
      <div class="card">
        <div class="p-3 text-center">
          <%= image_tag(item[:icon], class: 'fit-image-third card-img-top', alt: item[:icon]) %>
        </div>
        <div class="card-body">
          <h5 class="title"><%= item[:title] %></h5>
          <p class="card-text">
            <%= item[:description] %>
          </p>
        </div>
      </div>
    </div>
  <% end %>
```

The file es.yml, it could be conformed as follows:

```
es:
  hello: "Hello world"
  brightcoders:
    index:
      description_section:
        items:
        - title: "¿Qué somos?"
          description: "Somos una aceleradora de talento que brinda a jóvenes con iniciativa y ganas de aprender programas de
                    certificación diseñados para potenciar sus habilidades, mejorar sus posibilidades de acceso a un empleo de
                    calidad y condiciones de vida."
          icon: 'icon-cap'
        - title: "¿Qué ofrecemos?"
          description: "Una experiencia de aprendizaje única que te permitirá iniciar una carrera en tecnología gracias a la
                    capacitación y experiencia que lograrás colaborando en proyectos de desarrollo software reales y el
                    acompañamiento de una comunidad de mentores experimentados."
          icon: 'icon-computer'
        - title: "BrightCoders"
          description: "BrightCoders Academy capacita a estudiantes, egresados y jóvenes con ganas de aprender y con conocimientos
                    intermedios de programación orientada a objetos en HTML, CCS, y JS para iniciar una carrera en tecnología.
                    También es una opción para acreditar las prácticas o estancias profesionales."
          icon: 'icon-lamp'
```

This would allow for more BrightCoders description items to be added in the future.

Reply:
I have made this suggestion in this [PR](https://github.com/bright-coders/bc-landing-page-2021/pull/15).